apps/meteor/client/views/room/modals/FileUploadModal/FileUploadModal.tsx

```diff
@@ -101,11 +112,28 @@ const FileUploadModal = ({
 							</FieldRow>
 							{!name && <FieldError>{t('error-the-field-is-required', { field: t('Name') })}</FieldError>}
 						</Field>
+						{isImage && (
+							<Field>
+								<FieldLabel>{t('Upload_alt_description')}</FieldLabel>
+								<FieldRow>
+									<TextInput
+										value={description}
+										onChange={handleDescription}
+										placeholder={t('Upload_alt_description_placeholder')}
+										ref={ref}
+									/>
+								</FieldRow>
+							</Field>
+						)}
 						{showDescription && (
 							<Field>
-								<FieldLabel>{t('Upload_file_description')}</FieldLabel>
+								<FieldLabel>{t('Message')}</FieldLabel>
 								<FieldRow>
-									<TextInput value={description} onChange={handleDescription} placeholder={t('Description')} ref={ref} />
+									<Box is='section' display='flex' flexDirection='column' flexGrow={1} flexShrink={1} flexBasis='auto' height='full'>
+										<ComposerPopupProvider room={room}>
+											<TextMessageBox showFormattingTips rid={room._id} onChange={handleMessage} />
+										</ComposerPopupProvider>
+									</Box>
```

Comment:
The popup component is not showing **over** the other fields in this modal, it's actually making the modal grow for it to be displayed:

![Captura de tela de 2024-02-08 16-34-10](https://github.com/RocketChat/Rocket.Chat/assets/36537004/f08090e7-be1a-406d-a6fe-fe55e9674812)

Am I missing something?